Sources/IBPCollectionViewCompositionalLayout/IBPCollectionCompositionalLayoutSolver.m

```diff
@@ -76,7 +76,8 @@ - (void)solveGroup:(IBPNSCollectionLayoutGroup *)group forContainer:(IBPNSCollec
         interItemFixedSpacing = group.interItemSpacing.spacing;
     }
     CGFloat interItemFlexibleSpacing = 0;
-    if (group.interItemSpacing.isFlexibleSpacing) {
+    BOOL hasInterItemFlexibleSpacing = group.interItemSpacing.isFlexibleSpacing;
```

Comment:
Checking `interItemSpacing > 0` is wrong, flexible `interItemSpacing = 0` means  `> 0`, should check `isFlexibleSpacing`